examples/lce_map_tools/convert_lce.py

```diff
@@ -129,6 +129,5 @@ def convert_lce_map(filename, outname, n_channels=98, tpc_radius=15.5):
         bla = json.dumps(output)
         outfile.write(bla.encode())
 
-
-
-convert_lce_map("xy-lce-map2.5mm.dat", "s2_xy_patterns_map_XENON100_Xerawdp0.4.5.json.gz")
+		
```

Comment:
__[Flake8]__

indentation contains mixed spaces and tabs


<!-- Comment by [SideCI](https://sideci.com) -->